testcontainers-dapr/src/main/java/io/dapr/testcontainers/DaprContainer.java

```diff
@@ -287,6 +301,42 @@ protected void configure() {
       withNetwork(Network.newNetwork());
     }
 
+    if (configuration != null) {
```

Comment:
@cicoyle this block of code seems to be moved. If there is no change in the logic of the code, please revert back to its original place.